.github/workflows/ci.yml

```diff
@@ -87,7 +87,6 @@ jobs:
           directory: empty
           token: 87cefb17-c44b-4f2f-8b30-1fff5769ce46
```

Comment:
It's better to move this token to the config as it would be easier to keep it in one shared place.

Reply:
v4 introduced OIDC, which would only work in trusted contexts (which PRs from forks aren't). They have corner case handling for such PRs.
With v5, it's possible to toggle tokenless uploads in the Codecov UI. Not sure if this is an org-only setting or it could be per-repo. I haven't tested it.